Objects/typevarobject.c

```diff
@@ -458,6 +461,75 @@ unpack_typevartuples(PyObject *params)
     }
 }
 
+static PyObject *
+typeparam_reduce_anonymous(PyObject *self, PyObject *owner)
+{
+    assert(PyTuple_Check(owner) && PyTuple_GET_SIZE(owner) == 3);
```

Comment:
```suggestion
    assert(PyTuple_Check(owner));
    assert(PyTuple_GET_SIZE(owner) == 3);
```

To know which assertion is actually failing.